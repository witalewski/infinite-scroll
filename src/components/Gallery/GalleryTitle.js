import React from 'react';

export const GalleryTitle = ({ title }) => (
  <div className="row">
    <div className="col">
      <h1>{title}</h1>
    </div>
  </div>
);
