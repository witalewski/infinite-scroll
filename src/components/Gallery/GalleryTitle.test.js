import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { GalleryTitle } from './GalleryTitle';

describe('GalleryTitle', () => {
  it('matches snapshot', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<GalleryTitle title="Test Gallery" />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
