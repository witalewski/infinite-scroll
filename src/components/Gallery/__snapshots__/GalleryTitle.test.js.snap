// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GalleryTitle matches snapshot 1`] = `
<ForwardRef(render)
  className="row"
>
  <div
    className="col"
  >
    <h1
      className="title"
    >
      Test Gallery
    </h1>
  </div>
</ForwardRef(render)>
`;
